docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -500,28 +500,28 @@ initially in tree[I].Len; the codes are produced in tree[I].Code.
        code length:
 
 .KS
-          code = 0;
-          bl_count[0] = 0;
-          for (bits = 1; bits <= MAX_BITS; bits++) {
-             code = (code + bl_count[bits-1]) << 1;
-             next_code[bits] = code;
-          }
+         code = 0;
```

Comment:
I just removed the extra space for each paragraph. Now the number headings "1)", "2)", "3)", and the paragraph text, and the code are all aligned to 3-space tabs relative to the start of the line.